.github/workflows/backend-libraries/patch-generic.yml

```diff
@@ -0,0 +1,82 @@
+name: Patch
```

Comment:
The patch flow is not consistent with the release workflow, in the release workflow you don't create a new branch, you just push a tag on main, then the patch could never work.
I think you can do it more easily here, just take the exact same workflows as for backend-applications and just remove docker parts